src/base/rss/private/rss_parser.cpp

```diff
@@ -258,11 +257,15 @@ void Parser::parse_impl(const QByteArray &feedData)
         }
     }
 
-    if (xml.hasError())
-        m_result.error = xml.errorString();
-    else if (!foundChannel)
+    if (!foundChannel) {
         m_result.error = tr("Invalid RSS feed.");
-    else
+    }
+    else if (xml.hasError()) {
+        m_result.error = tr("%1 (line: %2, column: %3, offset: %4).")
+                .arg(xml.errorString().arg(xml.lineNumber())
```

Comment:
should it be `.arg(xml.errorString()).arg(xml.lineNumber())` ?

Reply:
Of course... Thank you!
Fixed now.